pkg/app/piped/executor/waitapproval/waitapproval.go

```diff
@@ -28,7 +29,8 @@ import (
 )
 
 const (
-	approvedByKey = "ApprovedBy"
```

Comment:
I think instead of introducing a new key `CurrentApprovers` to replace this, we have to store the approved user list into this old key because the web is still using this key to extract the configured list.